webpack.mix.js

```diff
@@ -63,6 +70,12 @@ mix.options({
   ],
 });
 
+mix.webpackConfig({
+  externals: {
+    'acf': 'acf',
```

Comment:
What does this do. add acf support to blade?

Reply:
No it makes the `window.acf` object importable. Instead of `window.acf.addAction()` you can do `import { addAction } from 'acf';`. Unnecessary maybe? Packages usually do the same for wp objects in Gutenberg.